lib/middleware/topics/single/topic-get.js

```diff
@@ -3,7 +3,7 @@
 const helpers = require('../../../helpers');
 
 module.exports = function getTopic() {
-  return (req, res) => helpers.topic.getById(req.params.topicId)
+  return (req, res) => helpers.contentfulEntry.getById(req.params.topicId)
```

Comment:
This is a little bit of a hack, to avoid more changes than we need currently in the topic helper. Looking up an `askChangeTopicBroadcast` id would work here but we'd be missing the `message` property used for a broadcast, which seems confusing.

Reply:
I also thought about Conversations always querying the `GET /contentfulEntries/:id` instead of `/topics/:id`, but there's something nice and logical about Conversations querying the `GET /topics/:id` resource to find reply templates. It also had me wondering about renaming the `templates` property as  `replies` in our API response, e.g. 
```
    {
      "id": "4nwTwvXmfuuYAGYgusGyyW",
      "name": "Test autoReplyBroadcast",
      "type": "autoReplyBroadcast",
      "createdAt": "2018-07-25T14:26:03.277Z",
      "updatedAt": "2018-07-26T16:57:36.050Z",
      "message": {
        "text": "Hello, how are you today?",
        "attachments": [
        "template": "autoReplyBroadcast"
      },
      "replies": {
        "autoReply": {
          "text": "I'm doing well, thank you.",
          "attachments": [],
          "template": "autoReply"
        }
    }
 }
```

Reply:
Minor. Can we add a `//TODO` comment on this hack if we think we are coming back and cleaning it up afterwards?

Reply:
Re: renaming the `templates` property as `replies`. I think renaming to `replyTemplates` could help clear up confusions about if by `template` we mean a content type template (as in the property inside the message object), or a reply template.

While we are in this piece of code. I would like to suggest calling that `template` property inside the `message` object `contentTypeTemplate` instead ? What do you think? 

Something like:
```
{
  "data": {
    "id": "4nwTwvXmfuuYAGYgusGyyW",
    "name": "Test autoReplyBroadcast",
    "type": "autoReplyBroadcast",
    "createdAt": "2018-07-25T14:26:03.277Z",
    "updatedAt": "2018-07-26T16:57:36.050Z",
    "message": {
      "text": "Hello, how are you today?",
      "attachments": [
        
      ],
      "contentTypeTemplate": "autoReplyBroadcast"
    },
    "replyTemplates": {
      "autoReply": {
        "text": "I'm doing well, thank you.",
        "attachments": [],
        "template": "autoReplyBroadcast"
      }
    }
  }
}
```

Reply:
This `message.template` property here is what gets saved to the outbound message `template` property in Conversations... https://github.com/DoSomething/gambit-conversations/blob/2.14.1/lib/middleware/messages/broadcast/broadcast-parse.js#L10, and meant to match the objects contained in the `templates` property, as they are the reply messages we send back.
I'm starting to second guess renaming `templates` to `replies` or `replyTemplates`, as it'd make sense to change them on the `topics` response.  The gotcha there is that the `web*` templates aren't really replies, and are used for the outbound web signup messages. Similarly, I imagined the reminder messaging fields would be embedded in the `templates` well .. these would outbound and wouldn't make sense to live in `replies`.